options/locale/locale_en-US.ini

```diff
@@ -988,6 +988,16 @@ file_view_rendered = View Rendered
 file_view_raw = View Raw
 file_permalink = Permalink
 file_too_large = The file is too large to be shown.
+bidi_bad_header = `This file contains unexpected BiDi Unicode characters!`
+bidi_bad_description = `This file contains unexpected BiDi Unicode characters that may be processed differently from what appears below.<br>If your use case is intentional and legitimate, you can safely ignore this warning.<br>Use the Escape button to reveal hidden characters.`
+bidi_bad_description_escaped = `This file contains unexpected BiDi Unicode characters.<br>Hidden unicode characters are escaped below.<br>Use the Unescape button to show how they render.`
```

Comment:
I think people not familiar with BiDi might have an easier time understanding these warnings if they said "bidirectional", which is self-explanatory.

Reply:
```suggestion
bidi_bad_header = `This file contains unexpected Bidrectional Unicode characters!`
bidi_bad_description = `This file contains unexpected Bidrectional Unicode characters that may be processed differently from what appears below.<br>If your use case is intentional and legitimate, you can safely ignore this warning.<br>Use the Escape button to reveal hidden characters.`
bidi_bad_description_escaped = `This file contains unexpected Bidrectional Unicode characters.<br>Hidden unicode characters are escaped below.<br>Use the Unescape button to show how they render.`
```